recirq/fermi_hubbard/execution.py

```diff
@@ -127,6 +127,10 @@ def cirq_resolvers(cls) -> Dict[str, Optional[Type]]:
     def _json_dict_(self):
         return cirq.dataclass_json_dict(self)
 
+    @classmethod
+    def _json_namespace_(cls):
+        return 'recirq'
```

Comment:
isn't this a breaking change? if we have experimental data saved without the recirq namespace?

Reply:
Hmmm... I am not sure.  Is there a way to resolve this somehow?  The json code requires a namespace since it is not a cirq class.  Would we have to put a custom resolver into cirq, or can we somehow put a custom resolver into reCirq?

Have we encountered this before?

Reply:
well we've never changed the requirements for 3rd party code before (here: the change is we need a namespace when that used to not be required)

there's already a custom resolver to resolve this class's name (without the namespace). You're right: we could fix the issue by resolving both the namespaced and un-namespaced version for backwards compatibility

Reply:
"we could fix the issue by resolving both the namespaced and un-namespaced version for backwards compatibility".  Do you know how to do this?  Can you point me in the right direction?

We probably should get this in, since I think there is a new batch of doc errors that is popping up, sigh...

Reply:
looks like @mrwojtek has put a `cirq_resolvers` classmethod on each of these objects that includes itself. In addition to `cls.__name__: cls` you'd add `f'recirq.{cls.__name__}: cls`

Reply:
Done.  I will commit this shortly so I can start to make progress on the doc failures in recirq.